backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/registerApplicationJavalinHandler.kt

```diff
@@ -0,0 +1,26 @@
+package app.ehrenamtskarte.backend.application.webservice
+
+import app.ehrenamtskarte.backend.auth.webservice.JwtService
+import io.javalin.Javalin
+import io.javalin.http.Context
+import java.io.File
+
+
+
+fun registerApplicationJavalinHandler(javalin: Javalin, applicationData: File) {
+    javalin.get("/application/:applicationId/file/:fileIndex") { ctx ->
+        if (JwtService.verifyRequest(ctx) !== null) {
+            val applicationId = ctx.pathParam("applicationId")
+            val fileIndex = ctx.pathParam("fileIndex");
+            val file = File(applicationData, "$applicationId/file/$fileIndex")
+            if (!file.isFile) {
+                ctx.status(404)
+            } else {
+                ctx.contentType("application/octet-stream")
+                ctx.result(file.inputStream())
+            }
+        } else {
+            ctx.status(404)
```

Comment:
```suggestion
            ctx.status(401)
```

Reply:
I wanted to make it impossible to check whether a file exists or not when you are not authenticated. Thats why I returned 404